docs/products/kafka/howto/change-retention-period.rst

```diff
@@ -0,0 +1,37 @@
+Change data retention period
+==============================
+
+To avoid running out of disk space, by default Apache Kafka drops the oldest messages from the beginning of each log after their retention period expires. **Aiven for Apache Kafka** allows you to configure the retention period for each topic.
+
+For a single topic
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Log in to the Aiven web console and select your service.
+
+#. Click the **Topics** tab and then click the topic that you want to modify.
+
+#. In the *Advanced configuration* view find **Retention ms**.
```

Comment:
Worth mentioning there is also Retention bytes if someone wants to fix the content based on size on disk rather than time